tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -89,6 +89,7 @@ declare namespace ts {
         namespace protocol {
             enum CommandTypes {
                 JsxClosingTag = "jsxClosingTag",
+                JsxMirrorCursor = "jsxMirrorCursor",
```

Comment:
If we want to align this with the LSP, we may instead want to call this `jsxLinkedEditingRange`